lib/internal/test_runner/test.js

```diff
@@ -455,4 +447,35 @@ class Test extends AsyncResource {
   }
 }
 
-module.exports = { kDefaultIndent, kSubtestsFailed, kTestCodeFailure, Test };
+class ItTest extends Test {
```

Comment:
We need a constructor here as long as V8 hasn't implemented https://github.com/tc39/ecma262/pull/2216.
```suggestion
class ItTest extends Test {
  constructor(opt) { super(opt); } // eslint-disable-line no-useless-constructor
```